modules/installation-configuration-parameters.adoc

```diff
@@ -1654,96 +1681,122 @@ In {product-title} 4.13, the following vSphere configuration parameters are depr
 The following table lists each deprecated vSphere configuration parameter:
 
 .Deprecated VMware vSphere cluster parameters
-[cols=".^2,.^3a,.^3a",options="header"]
+[cols=".^2a,.^4a,.^2a",options="header"]
 |====
 |Parameter|Description|Values
 
-|`platform.vsphere.apiVIP`
-|The virtual IP (VIP) address that you configured for control plane API access.
-a|An IP address, for example `128.0.0.1`.
+l|platform
+    vsphere
+      apiVIP
+a|The virtual IP (VIP) address that you configured for control plane API access.
```

Comment:
I have no idea if this is the issue you are hitting, but doesn't the `a` in the table spec (`^2a` for this column) make the `a`s down here irrelevant?

Reply:
Thanks, @jeana-redhat . I adopted your footnote solution and it works well. The last column is still expanding into the universe.

Reply:
Good thing the universe is also expanding to accommodate it, I suppose :sweat_smile: